app/controllers/main_controller.rb

```diff
@@ -6,10 +6,16 @@ def index
   def check
     @card = Card.where(original_text: params[:card][:original_text]).take
     translated_text = params[:card][:translated_text]
-    if @card.check_translation(translated_text)
+    check = @card.check_translation(translated_text)
+    if check[:success]
       flash[:notice] = "Correct"
     else
-      flash[:error] = "Wrong"
+      flash[:error] = %Q(Wrong:
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.